README.md

```diff
@@ -401,7 +401,17 @@ formatUnit(1000, 'meter', null, 'fr); // 1 000 mètres'
 ## How do I contribute?
 
 We appreciate your interest!  Please see the [Guidelines Repository](https://github.com/dojo/guidelines#readme) for the
-Contributing Guidelines and Style Guide.
+Contributing Guidelines.
+
+### Code Style
```

Comment:
Be good to get a specific review on the wording here.

Reply:
reads well to me